sources/packages/api/src-sql/MSFAANumbers/Add-col-offering-cancelled.sql

```diff
@@ -0,0 +1,27 @@
+-- Create offering_intensity
+ALTER TABLE
+    sims.msfaa_numbers
+ADD
+    COLUMN IF NOT EXISTS offering_intensity VARCHAR(10) NOT NULL DEFAULT 'Full Time';
+
+COMMENT ON COLUMN sims.msfaa_numbers.offering_intensity IS 'Offering intensity for the MSFAA Number Part Time/ Full Time.';
+
+ALTER TABLE
+    sims.msfaa_numbers
+ADD
+    COLUMN IF NOT EXISTS new_issuing_province VARCHAR(2);
+
+COMMENT ON COLUMN sims.msfaa_numbers.new_issuing_province IS 'MSFAA Number New issuing Province.';
+
+ALTER TABLE
+    sims.msfaa_numbers
+ADD
+    COLUMN IF NOT EXISTS cancelled_date DATE;
```

Comment:
is this a timestamp? or only date?

Reply:
just the date, similar to service_provider_received_date from the response file